lib/cryptoexchange/services/market.rb

```diff
@@ -1,6 +1,7 @@
 module Cryptoexchange
   module Services
     class Market
+      attr_reader :raw_response
```

Comment:
`raw_response` can be removed after https://github.com/coingecko/cryptoexchange/pull/252/files#r158582042